manifests/charts/ztunnel/values.yaml

```diff
@@ -43,13 +43,7 @@ multiCluster:
   # with Istiod configuration.
   clusterName: ""
 
-# meshConfig defines runtime configuration of components.
-# For ztunnel, only defaultConfig is used, but this is nested under `meshConfig` for consistency with other
-# components.
-# TODO: https://github.com/istio/istio/issues/43248
-meshConfig:
-  defaultConfig:
-    proxyMetadata: {}
-
-# Ambient redirection mode: "iptables" or "ebpf"
-redirectMode: "iptables"
+# For ztunnel, only defaultConfig in meshConfig is used. To avoid confusion from using the full meshConfig, we define a
+# proxyConfig here to replace the meshConfig.defaultConfig
+proxyConfig:
```

Comment:
maybe this should be under meshConfig

Reply:
This was discussed in https://github.com/istio/istio/pull/43584#discussion_r1117279399. It appears that only a few fields from MeshConfig may be utilized, so it would be preferable to set them individually rather than adding the entire structure?